airship_tempest_plugin/services/shipyard/json/document_staging_client.py

```diff
@@ -37,7 +38,10 @@ def get_configdocs_status(self):
         resp, body = self.get('configdocs')
         self.expected_success(200, resp.status)
         body = json.loads(body)
-        return rest_client.ResponseBody(resp, body)
+        if isinstance(body, list):
```

Comment:
is this needing when doing a get on a specific artifact/resource?

Reply:
This is returning a List only; therefore used "ResponseBodyList".